libstorage-ng.spec.in

```diff
@@ -15,246 +16,182 @@
-%{!?python_sitearch: %global python_sitearch %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib(1)")}
-%{!?py_requires: %global py_requires Requires: python}
-
-%if 0%{?ruby_sitelib} == 0
-%if 0%{?fedora_version} || 0%{?centos_version} || 0%{?rhel_version} || 0%{?fedora} || 0%{?rhel}
-%{!?ruby_sitelib: %global ruby_sitelib %(ruby -r rbconfig -e 'vd = Config::CONFIG["vendorlibdir"]; print(vd ? vd : Config::CONFIG["sitelibdir"])')}
-%{!?ruby_sitearch: %global ruby_sitearch %(ruby -r rbconfig -e 'vad = Config::CONFIG["vendorarchdir"]; print(vad ? vad : Config::CONFIG["sitearchdir"])')}
-%{!?ruby_vendor: %global ruby_vendor %(ruby -r rbconfig -e 'vad = Config::CONFIG["vendor"]; print(vad ? vad : Config::CONFIG["vendor"])')}
-%endif
-%endif
-
-%prep
-%setup -n libstorage-ng-%{version}
-
-%build
-export CFLAGS="$RPM_OPT_FLAGS -DNDEBUG"
-export CXXFLAGS="$RPM_OPT_FLAGS -DNDEBUG"
-
-aclocal
-libtoolize --force --automake --copy
-autoheader
-automake --add-missing --copy
-autoconf
-
-%{?suse_update_config:%{suse_update_config -f}}
-%if 0%{?fedora_version} || 0%{?centos_version}
-CXXFLAGS="-I/usr/include/libxml2" \
-%else
-%if 0%{?rhel_version} >= 500 && 0%{?rhel_version} < 600
-CXXFLAGS="-I/usr/include/libxml2 -I/usr/include/c++/4.1.1/tr1" \
-%endif
-%endif
+%package lang
+Summary:        Languages for package %{name}
+Group:          System/Localization
+Supplements:    packageand(bundle-lang-other:%{name})
+Provides:       %{name}-lang-all = %{version}
+BuildArch:      noarch
 
-./configure --libdir=%{_libdir} --prefix=%{prefix} --mandir=%{_mandir} --disable-silent-rules
-make %{?jobs:-j%jobs}
+%description lang
+Provides translations to the package %{name}
 
-%check
-make %{?jobs:-j%jobs} check LOCALEDIR=$RPM_BUILD_ROOT/usr/share/locale VERBOSE=1
-
-%install
-make install DESTDIR="$RPM_BUILD_ROOT"
-
-install -d -m 755 $RPM_BUILD_ROOT/run/libstorage
-touch $RPM_BUILD_ROOT/run/libstorage/lock
-
-%{find_lang} libstorage-ng
-
-rm -f $RPM_BUILD_ROOT/%{python_sitearch}/_storage.a
-rm -f $RPM_BUILD_ROOT/%{python_sitearch}/_storage.la
-%if 0%{?suse_version}
-%if "%{?rb_vendorarchdir}" != ""
-rm -f $RPM_BUILD_ROOT/%{rb_vendorarchdir}/storage.la
-%else
-rm -f $RPM_BUILD_ROOT/%{rb_vendorarch}/storage.la
-%endif
-%endif
-%if 0%{?mandriva_version}
-rm $RPM_BUILD_ROOT/%{ruby_sitearchdir}/storage.la
-%endif
-%if 0%{?fedora_version} || 0%{?rhel_version} || 0%{?centos_version}
-rm $RPM_BUILD_ROOT/%{ruby_sitearch}/storage.la
-%endif
-
-%clean
-rm -rf "$RPM_BUILD_ROOT"
-
-%package -n libstorage-ng@LIBVERSION_MAJOR@
-
-Requires:	coreutils
-Requires:	cryptsetup
-Requires:	device-mapper
-Requires:	dmraid
-Requires:	lsscsi >= 0.26
-Requires:	lvm2
-Requires:	mdadm >= 3.3
-Requires:	multipath-tools
-Requires:	parted >= 3.1
-Requires:	udev
-Requires:	util-linux >= 2.16
+%package -n %{libname}
+Summary:        Library for storage management
+Group:          System/Libraries
+Requires:       %{name}-lang
```

Comment:
This is usually not done as it was considered fine to just pull the language, of course if you want just add the = %{version}